### PR TITLE
Suppress submission form link on submission form

### PR DESCRIPTION
**Pull request by MatthewGidcomb**
It's not particularly useful and easy to confuse with the form submission.

Reported by @chriswininger

### CODE REVIEW

File: web-portal/layouts/default.vue [hunk 1]
```diff
@@ -14,6 +14,7 @@
     <!-- PopUps and Modals -->
     <!-- Regular Old Modal -->
     <ii-modal />
+    <!-- Phone-specifc call-to-action ("submit your event") -->
```
Comment (rlmead) [inline web-portal/layouts/default.vue]: extremely minor copy edit: `specifc` -> `specific`

File: web-portal/pages/subscribe.vue [hunk 2]
```diff
@@ -26,7 +26,7 @@
 
 <script>
   export default {
-    layout: 'no-subscribe'
+
```
Comment (rlmead) [inline web-portal/pages/subscribe.vue]: do we still need this script if it's empty?

**Review by rlmead — APPROVED** [review]

File: web-portal/pages/subscribe.vue (hunk 2 above)
Comment (MatthewGidcomb) [reply web-portal/pages/subscribe.vue]: Probably not.

File: web-portal/layouts/default.vue (hunk 1 above)
Comment (MatthewGidcomb) [reply web-portal/layouts/default.vue]: Yep, thanks.

**Pull request merged**